plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -337,6 +345,27 @@ def _check_update_trace_data(
                 ).loc[s_res.index]
             else:
                 trace["hovertext"] = hovertext
+
+            # Check if the marker size neets to be resampled
+            if not hasattr(trace, "marker"):
+                trace["marker"] = {}
+
+            marker_size = hf_trace_data.get("marker_size")
+            if isinstance(marker_size, (np.ndarray, pd.Series)):
+                trace["marker"]["size"] = self._to_hf_series(
+                    x=hf_trace_data["x"], y=marker_size
+                ).loc[s_res.index]
+            else:
+                trace["marker"]["size"] = marker_size
+
+            marker_color = hf_trace_data.get("marker_color")
+            if isinstance(marker_color, (np.ndarray, pd.Series)):
+                trace["marker"]["color"] = self._to_hf_series(
+                    x=hf_trace_data["x"], y=marker_color
+                ).loc[s_res.index]
+            else:
+                trace["marker"]["color"] = marker_color
```

Comment:
```py
            # Iterate over the high-frequency marker properties (stored in the 
            # _hf_data_container) and add resampled values to the trace (if necessary)
            hf_marker_props = ["size", "color"]
            for prop in hf_marker_props:
                marker_prop = hf_trace_data.get(f"marker_{prop}")
                if isinstance(marker_prop, (np.ndarray, pd.Series)):
                    trace["marker"][prop] = self._to_hf_series(
                        x=hf_trace_data["x"], y=marker_prop,
                    ).loc[s_res.index]
                else:
                    trace["marker"][prop] = marker_prop
```

Alternative implementation :arrow_up:
But tbh, I'm fine with keeping the original (more verbose) one